planet/cli/subscriptions.py

```diff
@@ -205,10 +205,10 @@ async def list_subscription_results_cmd(ctx,
     '--notifications',
     type=types.JSON(),
     help='Notifications JSON. Can be a string, filename, or - for stdin.')
-@click.option('--tools',
-              type=types.JSON(),
-              help='Toolchain JSON. Can be a string, filename, or - for stdin.'
-              )
+@click.option(
+    '--tools',
+    type=types.JSON(),
+    help='Toolchain JSON. Can be a string, filename, or - for stdin.')
```

Comment:
This is all on yapf and not explained by anything in its change log :unamused:

Reply:
And it's version dependent. With Python 3.7, yapf puts '--tools' on line 208. WTH.

Reply:
We're now using Python 3.8 for the CI linting checks.